src/Atmos/Model/AtmosModel.jl

```diff
@@ -52,6 +52,40 @@ struct AtmosModel{O,RS,T,M,P,R,SU,S,BC,IS} <: BalanceLaw
   init_state::IS
 end
 
+function AtmosModel{FT}(;orientation::O=FlatOrientation(),
+                         ref_state::RS=HydrostaticState(LinearTemperatureProfile(FT(200),
+                                                                                 FT(280),
+                                                                                 FT(grav) / FT(cp_d)),
+                                                                                 FT(0)),
+                         turbulence::T=SmagorinskyLilly{FT}(0.21),
+                         moisture::M=EquilMoist(),
+                         precipitation::P=NoPrecipitation(),
+                         radiation::R=NoRadiation(),
+                         subsidence::SU=NoSubsidence{FT}(),
+                         source::S=( Gravity(),
+                                     Coriolis(),
+                                     GeostrophicForcing{FT}(7.62e-5, 0, 0)),
+                         # TODO: Probably want to have different bc for state and diffusion...
+                         boundarycondition::BC=NoFluxBC(),
+                         init_state::IS=nothing) where {FT<:AbstractFloat,O,RS,T,M,P,R,SU,S,BC,IS}
+
```

Comment:
The defaults chosen in `LES_Configuration()` and `GCM_Configuration()` were discussed extensively with @akshaysridhar and were 'reasonable' for the two types of configurations. I suggest two constructors, with maybe something like:
```
abstract type AtmosConfiguration end
struct AtmosLESConfiguration <: AtmosConfiguration end
struct AtmosGCMConfiguration <: AtmosConfiguration end
function AtmosModel{FT}(::Type{AtmosLESConfiguration}; ...) ... end
function AtmosModel{FT}(::Type{AtmosGCMConfiguration}; ...) ... end
```

Reply:
Ah, that's exactly what we, thanks!